designs/2021-schema-object-rules/README.md

```diff
@@ -0,0 +1,287 @@
+- Repo: eslint/eslint and @eslint/eslintrc
+- Start Date: 2021-12-12
+- RFC PR: <https://github.com/eslint/rfcs/pull/85>
+- Authors: [bmish](https://github.com/bmish)
+
+# Require schemas and object-style rules
+
+## Summary
+
+<!-- One-paragraph explanation of the feature. -->
+
+We propose two changes to rules:
+
+- Requiring rules with options to specify schemas (removing support for rules with options that are missing schemas)
```

Comment:
This is a good change; a few of my plugins have rules that lack a schema because it’s a breaking change, but my planned workaround for those is to conditionally use an empty schema for eslint 9+